packages/rule-components/src/ReportDetails/RuleFeedback.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RuleFeedback = ruleId => (
```

Comment:
This should be object with `ruleId`
```suggestion
const RuleFeedback = ({ ruleId }) => (
```